server/posts/queries.ts

```diff
@@ -391,85 +302,84 @@ export const postNewContribution = async (
         is_new: true,
         is_own: true,
       },
-      boardSlug: board_slug,
+      boardSlug: threadData.board_slug,
     };
   };
-  return tx
-    ? createContribution(tx)
+  return transaction
+    ? createContribution(transaction)
     : pool.tx("create-contribution", createContribution);
```

Comment:
Wondering if it would be a good idea to indicate somewhere that `tx()` is a transaction since the amendments were made to our versions of `tx` and `t` to clearly indicate they're transactions

Reply:
Unfortunately that's just the name the library uses. The libraries I'm evaluating have better names for it.

Reply:
Yeah, I did skim the library to see if there was any alternative for it. Until we make a switch, if we don't add some kind of indicator somewhere or comment that `.tx()` is a transaction, it'll be good to remember this so we can let people who aren't familiar know that's what this is if it comes up